### DISCUSSION TITLE
Is there an opt-out switch for leading "./" with -x and -X?

### DISCUSSION THREAD

**Discussion by GlassGruber** [sharkdp/fd#1129] [Q&A] [answered]
I'm aware of https://github.com/sharkdp/fd/issues/760 and I'm absolutely in favor of security by default. 
I'm trying to see if there is an opt-out switch that I missed or similar.

I'm trying to run a quick batch archiving of files via 7z but is failing because of the leading `./`.
I have a bunch of folders with lots of files, I'm trying to make a single archive with all folders but containing only specific files

Command used
```
fd -j1 -t d -d 1 -x 7z a archive.zip "{}/file1.txt" "{}/file2.txt" 
```

Expected content of archvive.zip
```
dir1/file1.txt
dir1/file2.txt
dir2/file1.txt
dir2/file2.txt
dir3/file1.txt
dir3/file2.txt
...
```

But instead is always 
```
file1.txt
file2.txt
```

Because the leading dot is making a specific path (I guess?) so 7z is considering a pointer to a specific file rather than a relative path that would respect the folders structure in place.

I've tried with `--strip-cwd-prefix` but apparently is not working with `-x` and `-X`.

As a workaround I've printed the directories without leading dot with `--strip-cwd-prefix` to a file and used xargs but was wondering if there is a way to skip this extra step.

Thank you!

**Comment by tavianator** [comment, marked as answer]
Should work soon: https://github.com/sharkdp/fd/pull/1121

**Reply by GlassGruber** [reply]
Boy can't believe I missed that issue, I've searched I swear! 🙃 
Thank you for your answer.
